### PR TITLE
To do list enhancements

### PR DESCRIPTION
**Pull request by jwu910**
I think this should be every non-destructive change i've made to your base code. Current notable enhancements are auto focusing on input as you click add new item. Restrictions that do not add blank entries to list. and a Clear All items button.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,19 +6,32 @@
     <link rel="stylesheet" type="text/css" href="assets/css/toDo.css">
     <script type="text/javascript" src="assets/js/lib/jquery-3.1.1.min.js"></script>
     <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
```
Comment (rainrivas) [inline index.html]: You'll need to move the bootstrap CSS to be BEFORE the custom CSS or else the boot strap styles will be applied over custom ones.

**Review by rainrivas — CHANGES_REQUESTED** [review]
Correct boostrap css declaration.

**Comment by jwu910** [comment]
I moved it, and removed the send to phone button. Also added tab index 0 to icon for future keyboard accessibility.

**Review by rainrivas — APPROVED** [review]
Looks good, not too worried about losing strike through here since we'll just be deleting items we finish

**Comment by jwu910** [comment]
Good stuff, thanks!

**Pull request merged**